add new relese v0.6.0 (#4761)
DaoCloud/DaoCloud-docs: docs/en/docs/baize/intro/release-notes.md

This page lists the Release Notes for Intelligent Engine,
so that you can learn its evolution path and feature changes.

## 2024-06-30

### v0.6.0

#### Features

> - **New**: Support for creating `Code Server` type `Notebook`, providing a native `VS Code` development experience.

Reviewer: ```suggestion
- **New**: Support for creating `Code` type `Notebook`, providing a native `VS Code` development experience.
```

> - **New**: Support for quickly copying `Notebook`.
> - **New**: When selecting a work cluster, display the cluster's status information, making it unselectable if it is disconnected or offline.
> - **New**: When creating inference services, support using `vLLM` as the inference engine, exposing native `vLLM` capabilities.
> - **New**: When creating inference services, `vLLM` supports configuring `Lora` inference parameters.
> - **Optimization**: When creating a `Notebook`, the default queue priority is adjusted to `High`.

Reviewer 2: ```suggestion
- **New**: Support for quickly copying `Notebook`.
- **New**: When selecting a work cluster, display the cluster's status information, making it unselectable if it is disconnected or offline.
- **Added** support using `vLLM` as the inference engine, exposing native `vLLM` capabilities when creating inference services.
- **Added** `vLLM` supports configuring `Lora` inference parameters when creating inference services.
- **Optimized** the default queue priority `High` when creating a `Notebook`.
```
v0.6.0 风格跟以前不太一样了

#### Fixes

- **Fixed** the issue where `Python` version prompts permission problems in certain cases within environment management.
- **Fixed** the issue where the inference service does not support stopping during exceptions.

## 2024-04-30

### v0.4.0

#### Features

- **Added** `Notebook` now supports local SSH access, compatible with various development tools such as `Pycharm`, `VS Code`, etc.
- **Added** Upgrade `Notebook` image to support the built-in `CLI` tool `baizectl`, for command-line task submission and management.
- **Added** `Notebook` adds affinity scheduling strategy configuration.
- **Added** Distributed training tasks can now configure `SHM size` through the UI.
- **Added** One-click restart function for training tasks.

> - **Added** Model training tasks support custom cluster scheduler specification.**

Reviewer 2: ```suggestion
- **Added** Model training tasks support custom cluster scheduler specification.
```
